### PR TITLE
[chore] add tests for Java auto-instrumentation

### PR DESCRIPTION
**Pull request by atoulme**
**Description:**
Add tests with Apache Tomcat, sending spans every second.

### CODE REVIEW

File: functional_tests/testdata/java/Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (jvoravong) [inline functional_tests/testdata/java/Makefile:1]:
```suggestion
```

File: functional_tests/testdata/java/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Node.js test image
```
Comment (jvoravong) [inline functional_tests/testdata/java/README.md:1]:
```suggestion
# Java test image
```

**Review by jvoravong — APPROVED** [review]

**Comment by jvoravong** [comment]
I thought the test failures were just random, turns out they are reproducible. We'll have to fix these tests first.

**Review by jvoravong — CHANGES_REQUESTED** [review]
I thought the test failures were just random, turns out they are reproducible. We'll have to fix these tests first.

**Comment by atoulme** [comment]
> I thought the test failures were just random, turns out they are reproducible. We'll have to fix these tests first.

Working on it

**Comment by atoulme** [comment]
I have found that the docker image for java was not in a public repository, fixing that. Trying.

**Review by jvoravong — APPROVED** [review]

**Review by jinja2 — APPROVED** [review]

**Pull request merged**